client/components/Footer.js

```diff
@@ -30,7 +30,7 @@ const Footer = () => {
                     </ChakraLink>
                 </Flex>
             </Flex>
-            <Flex flexDirection="row" justifyContent="space-between" alignItems="center" paddingMargin={1}>
+            <Flex flexDirection="row" justifyContent="space-between" alignItems="center" paddingmargin={1}>
```

Comment:
React keeps throwing an error message for this. This change gets rid of that error.